documentation/smg/11_ISIS_release.md

```diff
@@ -1,27 +1,43 @@
 # ISIS Release Process
 
-Horace is deployed separately to a number of systems across ISIS
+Horace deployed on iDAAaS system and machines dedicated for excitations as a single git repository available 
+read-only for excitation machine users and providing read/write access for the members of the excitations group.
 
-## ISIS Compute
+As the single place of the Horace code is available to all machines, the changes to the code made by releasers are immediately available to all iDAAaS machines.
 
-#### Contact
+## Horace installation in ISIS
+
+Common path `/usr/local/mprogs/Users` is added to the search path of all MATLAB sessions available to users processing the results of inelastic neutron scattering experiments in ISIS.
+ 
+On the machines, managed by excitation group this is achieved by adding this path to the end of the `{MATLAB Installation Folder}/toolbox/local/pathdef.m` file, though recent iDAAaS virtual machines seems use different approach for adding this path to the MATLAB search path.
+
+This path contains Horace initialization script `horace_on.m` which, on execution, adds all necessary folders to MATLAB search path and initialize all other Horace variables necessary for Horace operations. (i.e. Horace configurations). This folder also contains the worker script, which invokes `horace_on` script and initializes Horace for all parallel MATLAB sessions. 
```

Comment:
```suggestion
This path includes the Horace initialization script `horace_on.m` which, on execution, adds all necessary folders to MATLAB search path and initializes all other Horace variables necessary for Horace operations. (i.e. Horace configurations). This folder also contains the worker script which invokes the `horace_on` script and initializes Horace for all parallel MATLAB sessions. 
```
Did you mean i.e. so only configurations are initialized or did you mean e.g. so other things are initialized as well?

Reply:
should be e.g. indeed.